spec/requests/smoke/health_check_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "system health check spec", :smoke do
+  describe "GET /healthcheck" do
```

Comment:
Is there a difference between the tests here and the existing tests?

https://github.com/DFE-Digital/teacher-training-api/blob/711-healthcheck-smoke-test/spec/requests/heartbeat_spec.rb#L12

Reply:
Yes, this one does not stub out any data whereas the other tests different scenarios based on different errors which **could** occur

Reply:
Aaahh. *sound-of-penny-dropping* ... this is only run when the smoke tests run. Makes sense now. 👍